packages/protocol/contracts/common/GasPriceMinimum.sol

```diff
@@ -47,51 +49,48 @@ contract GasPriceMinimum is Ownable, Initializable, UsingRegistry {
   function initialize(
     address _registryAddress,
     uint256 initialGas,
-    uint256 targetDensityNumerator,
-    uint256 targetDensityDenominator,
-    uint256 adjustmentSpeedNumerator,
-    uint256 adjustmentSpeedDenominator,
-    uint256 infrastructureFractionNumerator,
-    uint256 infrastructureFractionDenominator
+    int256 _targetDensity,
+    int256 _adjustmentSpeed,
+    int256 _infrastructureFraction
   )
     external
     initializer
   {
     _transferOwnership(msg.sender);
     setRegistry(_registryAddress);
     gasPriceMinimum = initialGas;
-    setTargetDensity(targetDensityNumerator, targetDensityDenominator);
-    setAdjustmentSpeed(adjustmentSpeedNumerator, adjustmentSpeedDenominator);
-    setInfrastructureFraction(infrastructureFractionNumerator, infrastructureFractionDenominator);
+    setTargetDensity(_targetDensity);
+    setAdjustmentSpeed(_adjustmentSpeed);
+    setInfrastructureFraction(_infrastructureFraction);
   }
 
   /**
    * @notice Set a multiplier that impacts how quickly gas price minimum is adjusted.
    * @dev Value is expected to be < 1.
    */
-  function setAdjustmentSpeed(uint256 numerator, uint256 denominator) public onlyOwner {
-    require(denominator > 0 && numerator < denominator);
-    adjustmentSpeed = FractionUtil.Fraction(numerator, denominator);
-    emit AdjustmentSpeedSet(numerator, denominator);
+  function setAdjustmentSpeed(int256 _adjustmentSpeed) public onlyOwner {
+    require(_adjustmentSpeed < FixidityLib.fixed1());
```

Comment:
Hmm, not sure if this is the sort of thing to make into a modifier. 

One thing that I was thinking about is defining a constant `FIXED1` instead of calling `FixidityLib.fixed1()` everywhere, which should reduce both bytecode and gas costs. Thoughts?

Reply:
I just thought for legibility a modifier like `isProperFraction` could be nice

I think the FIXED1 or FIXED_ONE is a great idea